scripts/build-engine/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@cocos/build-engine",
-  "version": "4.3.17",
+  "version": "4.3.17-oh.1",
```

Comment:
The same as above.

Reply:
please cherry-pick this PR https://github.com/qiuguohua/cocos-engine/pull/9/files

Reply:
please cherry-pick these commits https://github.com/qiuguohua/cocos-engine/pull/9/files